tests/test_rand.py

```diff
@@ -8,3 +13,51 @@ def test_random():
         assert 0 <= n <= 2 ** 64 - 1
         assert n not in m
         m.add(n)
+
+
+def test_not_fork_safe():
+    q = Queue()
+    pid = os.fork()
+    if pid > 0:
+        # parent
+        rngs = {_rand.rand64bits() for i in range(100)}
+        child_rngs = q.get()
+        assert rngs == child_rngs
+    else:
+        # child
+        rngs = {_rand.rand64bits() for i in range(100)}
+        q.put(rngs)
+
+        # Kill the process so it doesn't continue running the rest of the
```

Comment:
subprocess test?

Reply:
Hmmm. I want to be the one explicitly forking and handling the fork. Subprocess test might obscure that a bit.

Reply:
yeah, I meant more if the entire code runs in a subprocess makes it easier to cleanup?

have the subprocess do the forking?